extensions/default/src/ViewerLayout/index.tsx

```diff
@@ -1,12 +1,33 @@
-import React, { useEffect, useState, useCallback } from 'react';
+import React, { useEffect, useState, useCallback, useLayoutEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 
 import { LoadingIndicatorProgress, InvestigationalUseDialog } from '@ohif/ui';
 import { HangingProtocolService, CommandsManager } from '@ohif/core';
 import { useAppConfig } from '@state';
 import ViewerHeader from './ViewerHeader';
 import SidePanelWithServices from '../Components/SidePanelWithServices';
-import { Onboarding } from '@ohif/ui-next';
+import { Onboarding, ResizablePanelGroup, ResizablePanel, ResizableHandle } from '@ohif/ui-next';
+
+// Id needed to grab the panel group for converting pixels to percentages
+const viewerLayoutResizablePanelGroupId = 'viewerLayoutResizablePanelGroup';
+
+// Ids needed for conditionally rendered resizable panels.
+// See https://github.com/bvaughn/react-resizable-panels/tree/main/packages/react-resizable-panels#how-can-i-fix-layoutsizing-problems-with-conditionally-rendered-panels.
+const viewerLayoutResizableLeftPanelId = 'viewerLayoutResizableLeftPanel';
+const viewerLayoutResizableRightPanelId = 'viewerLayoutResizableRightPanel';
+const viewerLayoutResizableViewportGridPanelId = 'viewerLayoutResizableViewportGridPanel';
```

Comment:
I don't need they are reused, can you just hardcode it below

Reply:
Certainly.